.buildkite/x-pack/elastic-agent/pipeline.xpack.elastic-agent.package.yml

```diff
@@ -0,0 +1,6 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+# This pipeline is only for 7.17 branch. See catalog-info.yml
+steps:
+  - label: "Example test"
+    command: echo "Hello!"
```

Comment:
Was it forgotten here? Or is it here as a placeholder that `catalog-info.yml` can reference?

Reply:
That's right